lib/app.js

```diff
@@ -87,7 +87,14 @@ class Mali extends Emitter {
       if (typeof path === 'object' && path.root && path.file) {
         protoFilePath = path.file
         if (!loadOptions.includeDirs) {
-          loadOptions.includeDirs = [path.root]
+          /**
+           * To support multiple folders to feed the protoLoader.
+           */
+          if (path.root instanceof Array) {
+            loadOptions.includeDirs = path.root
+          } else {
+            loadOptions.includeDirs = [path.root]
+          }
```

Comment:
```suggestion
          loadOptions.includeDirs = Array.isArray(path.root) ? path.root : [path.root]
```

Reply:
And can you also update the TypeScript and the JSDoc documentations?

Reply:
adding tests and documentation for sure!

Reply:
@anonrig Just pushed the change with one line and also added some tests. Let me know if it's ok.
Thanks for the fast feedback!